vanilla-app/package.json

```diff
@@ -14,15 +14,20 @@
     "vite": "4.3.9"
   },
   "dependencies": {
-    "@justeat/pie-design-tokens": "5.7.0",
-    "@justeattakeaway/pie-button": "0.34.0",
-    "@justeattakeaway/pie-card-container": "0.11.0",
-    "@justeattakeaway/pie-cookie-banner": "0.7.0",
-    "@justeattakeaway/pie-css": "0.6.0",
-    "@justeattakeaway/pie-divider": "0.8.0",
-    "@justeattakeaway/pie-icon-button": "0.18.0",
-    "@justeattakeaway/pie-icons-webc": "0.6.0",
-    "@justeattakeaway/pie-modal": "0.28.0",
-    "@justeattakeaway/pie-toggle-switch": "0.14.0"
+    "@justeat/pie-design-tokens": "5.8.2",
```

Comment:
any reason not to use latest for this package?

Reply:
Short answer is no, there wasn't any reason 😄